app/frontend/src/elm/Util/Formatter.elm

```diff
@@ -43,3 +44,17 @@ percentageConverter numerator denominator =
 formatEosQuantity : String -> String
 formatEosQuantity =
     String.toFloat >> Result.withDefault 0 >> Round.round 4
+
+
+
+-- Time
+
+
+timeFormatter : String -> String
```

Comment:
You may want to test this utility function :)

Reply:
Sure :)